graphics/generate-asciinema-gif

```diff
@@ -36,34 +36,34 @@ spoonfeed() {
 
 function script() {
   spoonfeed 2 'git ma~chete dis~cover --ch~ecked-out-since="2 weeks ago"'
-  spoonfeed 3 'y^'
-  spoonfeed 3 '~clear'
+  spoonfeed 5 'y^'
+  spoonfeed 1 '~clear'
 
   spoonfeed 1 'git ma~chete st~atus --l~ist-commits'
-  spoonfeed 4 '~clear'
+  spoonfeed 5 '~clear'
 
   spoonfeed 1 'git chec~kout d~evelop'
   spoonfeed 1 'git ma~chete tr~averse'
-  spoonfeed 3 'y^'
-  spoonfeed 3 'yq^'
-  spoonfeed 2 '~clear'
+  spoonfeed 4 'y^'
+  spoonfeed 4 'yq^'
+  spoonfeed 3 '~clear'
 
   spoonfeed 1 'git ma~chete st~atus --l~ist-commits'
-  spoonfeed 4 '~clear'
+  spoonfeed 5 '~clear'
 
   spoonfeed 1 'git ma~chete tr~averse'
-  spoonfeed 3 'y^'
-  spoonfeed 3 'yq^'
+  spoonfeed 4 'y^'
+  spoonfeed 2 'yq^'
   spoonfeed 5 '~clear'
   spoonfeed 1 '~exit'
 }
 
 init_contents='set -e -o pipefail -u; PS1=\(\\\$\(date\ +%H:%M:%S\)\)\ \$\ ; export GIT_MACHETE_DIM_AS_GRAY=true; export GIT_SEQUENCE_EDITOR=:'
 nested_command="bash --init-file <(echo '$init_contents')"
 script | asciinema rec --command="bash -c \"$nested_command\"" "$cast"
-# `head -n -3` to remove the 3 final lines responsible for the terminating `exit`
+# `head -n $(( $(wc -l "$cast" | awk '{print $1}') - 2 ))` to remove the 3 final lines responsible for the terminating `exit`
 # Note: for some reason, the resulting gif manifested weird behavior for widths lower than 120;
 # I needed to crop the right margin in an external tool.
```

Comment:
Oh! indeed, could you crop the right margin, so that the width is the same as it used to be (710px)? would be :benissimo: to automate that, should be rather easy/portable with `ImageMagick`

Reply:
Actually... `asciicast2gif` is now deprecated, the repo recommends using a newer tool, [`agg`](https://github.com/asciinema/agg). Could you check? maybe it no longer has this problem with widths>=120 required (and hence no postprocessing/cropping is required)

Reply:
I replaced `asciicast2gif` with `agg` but still had to crop the gif but now it is done automatically within the script.